smalldata_tools/lcls1/hutch_default.py

```diff
@@ -95,7 +57,21 @@ def sxrDetectors(beamCodes=[[162],[91]]):
 def xppDetectors(beamCodes=[[-137],[91]], env=None):
     dets=[]
     dets.append(lightStatus(codes=beamCodes))
-    dets.append(epicsDetector(PVlist=['att_T', 'att_T3rd', 'slit_s1_hw', 'slit_s1_vw', 'slit_s2_hw', 'slit_s2_vw', 'slit_s3_hw', 'slit_s3_vw', 'slit_s4_hw', 'slit_s4_vw', 'lxt_vitara', 'lxt', 'lxt_ttc', 'lxe', 'ccm_E', 'lom_E', 'lom_EC', 'gon_v', 'gon_h', 'gon_r', 'gon_x', 'gon_y', 'gon_z', 'gon_roll', 'gon_pitch', 'gon_kappa_eta', 'gon_kappa_kappa', 'gon_kappa_phi', 'gon_kappa_samx','gon_kappa_samy', 'gon_kappa_samz', 'gon_sam_phi', 'gon_sam_z', 'robot_x', 'robot_y', 'robot_z', 'robot_rx', 'robot_ry', 'robot_rz', 'robot_azi', 'robot_ele', 'robot_rad', 'las_comp_wp', 'las_opa_wp', 'las_drift_correction']))
+    dets.append(epicsDetector(PVlist=[
```

Comment:
For formatting changes like this do you want to just apply a formatter to avoid going back and forth on asthetic changes (`black`, `autopep8`, etc.)?

Reply:
Good idea, I will explore these options. I am a little worried it will break some things given how non-compliant this code is though...